model-optimizer/extensions/back/ConvolutionNormalizer.py

```diff
@@ -247,9 +247,6 @@ def replace_pattern(self, graph: Graph, match: dict):
             shape_src.connect(ss_0.in_port(0))
             ss_0.out_port(0).connect(node.in_port(2))
 
-            # Specification: *padding amount* is deduced from relation of input and output spatial shapes
```

Comment:
Is this correct approach? We have discussed that the fix should be done in the IE IR reader.

@slyalin , @GlebKazantaev what do you think?

Reply:
According to conversation with @ilyachur we have to setting all attributes in IR. So we need to fix it on MO side.

Reply:
@lazarevevgeny If I remember right we had a discussion, that in IR we should always set all attributes.

Reply:
@ilyachur, according to the documentation this attribute must be omitted if the padding type is "auto".

Reply:
Just have a quick call with @ilyachur  and he agreed that this is an IE Reader bug and MO should not be changed.